doc/docs/2.3.x/how-tos/basic-data-operations/sql-ingest.md

```diff
@@ -1,251 +1,232 @@
-!!! Example 
+1. Copy the following:
+    ```shell
+    kubectl create secret generic yourSecretName --from-literal=PACHYDERM_SQL_PASSWORD=yourDatabaseUserPassword --dry-run=client --output=json > yourSecretFile.json
+    ```
+2. Swap out `yourSecretName`, `yourDatabaseUserPassword`, and `yourSecretFile` with relevant inputs.
+3. Open a terminal and run the command.
+4. Copy the following: 
+    ```shell
+    pachctl create secret -f yourSecretFile.json
+    ```
+5. Swap out `yourSecretfile` with relevant filename. 
+6. Run the command. 
+7. Confirm secret by running `pachctl list secret`. 
+   
+!!! Note
+     Not all secret formats are the same. For a full walkthrough on how to create, edit, and view different types of secrets, see [Create and Manage Secrets in Pachyderm](../../advanced-data-operations/secrets/#create-a-secret).
 
-    In this example, we are leveraging Snowflake's support for queries traversing semi-structured data (here, JSON).
+### 2. Create a Database Connection 
 
-    - Find the [documentation for the support of semi-structured data in Snowflake](https://docs.snowflake.com/en/user-guide/semistructured-concepts.html#:~:text=Snowflake%20provides%20native%20support%20for,fast%20and%20efficient%20SQL%20querying.){target=_blank} here. 
+Pachyderm's SQL Ingest requires a connection string defined as a [Jsonnet URL parameter](#url-parameter-details) to connect to your database; the URL is structured as follows:
 
-    - The query in the following example will use the WEATHER schema in the public test database SNOWFLAKE_SAMPLE_DATA in the COMPUTE_WH warehouse. The column V of the table DAILY_14_TOTAL stores JSON files.
+```
+<protocol>://<username>@<host>:<port>/<database>?<param1>=<value1>&<param2>=<value2>
+```
 
-        Note the references to the JSON dataset elements by their hierarchical paths in the query:
+### 3. Create a Pipeline Spec 
+
+Pachyderm provides a [default Jsonnet template](https://raw.githubusercontent.com/pachyderm/pachyderm/{{ config.pach_branch }}/src/templates/sql_ingest_cron.jsonnet) that has key parameters built in. To use it, you must pass an argument for each [parameter](#parameters). 
+
+1. Copy the following:
+   ```shell
+   pachctl update pipeline --jsonnet https://raw.githubusercontent.com/pachyderm/pachyderm/{{ config.pach_branch }}/src/templates/sql_ingest_cron.jsonnet \
+     --arg name=<outputRepoName> \
+     --arg url="<connectionStringToDdatabase>" \
+     --arg query="<query>" \
+     --arg hasHeader=<boolean> \
+     --arg cronSpec="<pullInterval>" \
+     --arg secretName="<youSecretName>" \
+     --arg format=<CsvOrJson> 
+   ```
+2. Swap out all of the parameter values with relevant inputs. 
+3. Open terminal.
+4. Run the command.
+
+### 4. View Query & Results 
+
+- **To View Query String**: `pachctl inspect pipeline pipelineName`
+- **To View Output File Name**: `pachctl list file outputRepoName@master`
+- **To View Output File Contents**: `pachctl get file myingest@master:/0000` 
```

Comment:
```suggestion
- **To View Output File Contents**: `pachctl get file <pipelineName>@master:/0000` 
```

I think it's important to show in this new design that you can just refer to the pipeline name for most operations, since there is only one pipeline now.